.github/workflows/publish.yml

```diff
@@ -34,10 +49,25 @@ jobs:
           GITHUB_ACTIONS_REPO: ${{ github.repository }}
           GITHUB_ACTIONS_RUN_ID: ${{ github.run_id }}
           GITHUB_USERNAME_TOKEN: ${{ secrets.SONARIO_GH_TOKEN }}
-      - name: Verify - Published Charts
+
+      - name: Publish & Deploy Charts 🚀
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.SONARIO_GH_TOKEN }}
+          publish_dir: /tmp/helm/publish/helm-charts
+          enable_jekyll: false
+          keep_files: true
+          user_name: "Github-Actions-CI"
+          user_email: ${{env.GITHUB_USERNAME}}@users.noreply.github.com
```

Comment:
why that email?

Reply:
This is the syntax that we used earlier, looks like people use this email type when they are using a bot to push / publish.